taskcluster/ci/startup-test/kind.yml

```diff
@@ -36,12 +36,13 @@ task-defaults:
     run-on-tasks-for: []
     attributes:
         build-type: nightly
+        shipping-product: fenix
         # cannot safely run this in Firebase
         # turning off til we can safely run test in taskcluster
         # https://github.com/mozilla-mobile/fenix/issues/7702
         #nightly: true
     dependencies:
-        signing: signing-apk-fenix-nightly
+        signing: signing-apk-fenix-nightly-firebase
```

Comment:
I confirm this build flavor exists 👌

https://github.com/mozilla-mobile/firefox-android/blob/795f3950b671b102de99b47684a7089ffef10b96/taskcluster/ci/build-apk/kind.yml#L271